src/main/java/Ui.java

```diff
@@ -0,0 +1,62 @@
+import java.util.ArrayList;
+
+public class Ui {
+    public Ui() {
+
+    }
+
+    public static void printHello() {
+        String logo = "      ____        _        \n"
+                + "     |  _ \\ _   _| | _____ \n"
+                + "     | | | | | | | |/ / _ \\\n"
+                + "     | |_| | |_| |   <  __/\n"
+                + "     |____/ \\__,_|_|\\_\\___|\n";
+        String greetingText = "    ____________________________________________________________\n" +
+                logo +
+                "     Hello! I'm Duke\n" +
+                "     What can I do for you?\n" +
+                "    ____________________________________________________________";
```

Comment:
Maybe you can use a String variable to store the lines. It may save more space and look nicer :)

Reply:
Also, you can use a final constant string value here as well. This string will not be changed.